README.md

```diff
@@ -659,6 +659,10 @@ Contributors: Timon Kanters, Jeroen Erik Jensen, Krzysztof Karczewski
 
 ## Release notes
 
+### 1.6.5
```

Comment:
Since this change break the API, it should probably be 1.7.0.

Reply:
Done